.github/workflows/integrationTest.yml

```diff
@@ -214,6 +226,49 @@ jobs:
           cd buildMSI
           .\create_msi.ps1 ${{ github.sha }} ${{ secrets.S3_INTEGRATION_BUCKET }}
 
+  #GH actions set up gpg only works on ubuntu as of this commit date
+  GPGSignMacAndWindowsPackage:
+    name: 'SignMacAndWindowsPackage'
+    runs-on: ubuntu-latest
+    needs: [BuildMSI, MakeMacPkg]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-gpg@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.TERRAFORM_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.TERRAFORM_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+
+      - name: Cache sig
+        id: cached_sig
+        uses: actions/cache@v2
+        with:
+          key: "cached_sig_${{ github.sha }}"
+          path: packages/amazon-cloudwatch-agent.msi.sig
```

Comment:
since we're not using this file path, can we change this to cache go.mod for consistency